src/Sylius/Bundle/CoreBundle/Templating/Helper/PriceHelper.php

```diff
@@ -57,10 +57,21 @@ public function getLowestPriceBeforeDiscount(ProductVariantInterface $productVar
         Assert::keyExists($context, 'channel');
         Assert::isInstanceOf($this->productVariantPriceCalculator, ProductVariantPricesCalculatorInterface::class);
 
-        return $this
-            ->productVariantPriceCalculator
-            ->calculateLowestPriceBeforeDiscount($productVariant, $context)
-        ;
+        if (\method_exists($this->productVariantPriceCalculator, 'calculateLowestPriceBeforeDiscount')) {
+            return $this
+                ->productVariantPriceCalculator
+                ->calculateLowestPriceBeforeDiscount($productVariant, $context)
+            ;
+        }
+
+        trigger_deprecation(
+            'sylius/sylius',
+            '1.13',
+            'Not having `calculateLowestPriceBeforeDiscount` method on %s is deprecated since Sylius 1.13 and will be required in Sylius 2.0.',
+            $this->productVariantPriceCalculator::class,
+        );
+
+        return $this->getPrice($productVariant, $context);
```

Comment:
Logically speaking, if lowest price before discount doesn't exists, then it should return `null`, since method return allows this as well.

But if it works alright for the admin panel, then it's probably alright, but I'm not familiar with this overall feature yet.